.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -16,20 +16,26 @@ about: If something isn't working as expected 🤔
 ### Issue Description
 <!-- Please include what's happening, expected behavior, and any relevant code samples -->
 
-##### Complete output when running fastlane, including the stack trace and command used
-<!-- You can use: `--capture_output` as the last commandline argument to get that collected for you -->
+##### Command executed
+<!-- The command you executed on the command line, that resulted in an error -->
 
-<!-- The output of `--capture_output` could contain sensitive data such as application ids, certificate ids, or email addresses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->
+##### Complete output when running fastlane, including the stack trace and command used
+<!-- 
+You can use `--capture_output` as the last commandline argument for many commands to get that collected for you. Otherwise, please do it manually.
```

Comment:
Space inside `commandline`.

Reply:
```suggestion
You can use `--capture_output` as the last command line argument for many commands to get that collected for you. Otherwise, please do it manually.
```